src/csvgrep.c

```diff
@@ -144,6 +144,9 @@ static void print_help() {
     fprintf(stderr, "-s ,\n");
     fprintf(stderr, "\tWhich character to use as separator (default is ,)\n");
     fprintf(stderr, "-p column/pattern/\n");
+    fprintf(stderr, "\tColumn is : \n");
+    fprintf(stderr, "\t\t- the column name as specified in header line \n");
+    fprintf(stderr, "\t\t- or the column index prefixed with # (eg: #0) \n");
```

Comment:
👍 thanks for adding clear description, maybe make explicit that the first column is index by 0?

Reply:
Agree, I'll do it